packages/astro/test/redirects.test.js

```diff
@@ -68,10 +75,31 @@ describe('Astro.redirect', () => {
 				const response = await app.render(request);
 				expect(response.status).to.equal(308);
 			});
+
+			it('Forwards params to the target path - single param', async () => {
+				const app = await fixture.loadTestAdapterApp();
+				const request = new Request('http://example.com/source/x');
+				const response = await app.render(request);
+				expect(response.headers.get("Location")).to.equal("/not-verbatim/target1/x");
+			});
+
+			it('Forwards params to the target path - multiple params', async () => {
+				const app = await fixture.loadTestAdapterApp();
+				const request = new Request('http://example.com/source/x/y');
+				const response = await app.render(request);
+				expect(response.headers.get("Location")).to.equal("/not-verbatim/target2/x/y");
+			});
+
+			it('Forwards params to the target path - spread param', async () => {
+				const app = await fixture.loadTestAdapterApp();
+				const request = new Request('http://example.com/source/x/y/z');
+				const response = await app.render(request);
+				expect(response.headers.get("Location")).to.equal("/not-verbatim/target3/x/y/z");
+			});
 		});
 	});
 
-	describe('output: "static"', () => {
+	describe.skip('output: "static"', () => {
```

Comment:
Is it intended? If so, could you add a comment or `TODO` to tell us when we should unskip it?